test_statsfile/BUILD

```diff
@@ -0,0 +1,25 @@
+load(
+    "//scala:scala.bzl",
+    "scala_binary",
+)
+load("//test:check_statsfile.bzl", "check_statsfile", "check_statsfile_empty")
+
+scala_binary(
+    name = "Simple",
+    srcs = ["Simple.scala"],
+    main_class = "scalarules.test.Simple",
+    print_compile_time = True,
+    deps = [],
+)
+
+scala_binary(
+    name = "SimpleNoStatsFile",
+    srcs = ["Simple.scala"],
+    main_class = "scalarules.test.Simple",
+    print_compile_time = True,
+    deps = [],
+)
```

Comment:
how are these two targets different in terms of whether statsfile is enabled? I am unable to spot the difference.

Reply:
`check_statsfile`/`check_statsfile_empty` expand into genrules and if the same name is use they would collide